ibex_core.core

```diff
@@ -39,7 +40,10 @@ filesets:
       - rtl/ibex_register_file_ff.sv # generic FF-based
       - rtl/ibex_core.sv
       - rtl/ibex_pointer_authentication.sv
-      - rtl/ibex_cipher.sv
+      - rtl/gift.sv
+      - rtl/gift_middle_rounds.sv
+      - rtl/gift_round.sv
+      - rtl/gift_update_key.sv
```

Comment:
I would also add a separate core file for these files and make `ibex_core.core` depend on this.

Reply:
Yes, I moved it to `gift_cipher.core`.